ftw/linkchecker/command/broken_link.py

```diff
@@ -28,12 +29,22 @@ def __init__(self):
         self.response_time = 'Unknown response time'
         self.error_message = 'No error occurred'
         self.creator = 'Unknown creator'
+        self.source_state = 'Unknown status'
```

Comment:
`Unknown status` is not a good default value, as an object may not a have workflow assigned and thus does not have a workflow state. So a better default would be an empty string.

Reply:
> Could you also add some tests, please?

There are tests. What else would you like to be tested?

Reply:
> `Unknown status` is not a good default value, as an object may not a have workflow assigned and thus does not have a workflow state. So a better default would be an empty string.

I'll change that then.